src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/ReferencedPackageModel.cs

```diff
@@ -12,14 +12,14 @@
 
 namespace NuGet.PackageManagement.UI.Models
 {
-    public class ReferencedPackageModel : PackageModel, IVulnerable
+    public class ReferencedPackageModel : PackageModel, IVulnerableCapable
     {
-        private readonly IVulnerable _vulnerableCapability;
+        private readonly IVulnerableCapable _vulnerableCapability;
 
         public ReferencedPackageModel(
             PackageIdentity identity,
             string packagePath,
-            IVulnerable vulnerabilityCapability,
+            IVulnerableCapable vulnerabilityCapability,
```

Comment:
nit: for consistency

```suggestion
            IVulnerableCapable vulnerableCapability,
```